scripts/plotting/global_latlon_map.py

```diff
@@ -65,6 +66,15 @@ def global_latlon_map(adfobj):
     #CAM simulation variables (this is always assumed to be a list):
     case_names = adfobj.get_cam_info("cam_case_name", required=True)
 
+    if len(case_names) > 1:
+        #Check if multi-plots are desired from yaml file
+        if adfobj.read_config_var('multi_case_plots'):
+            multi_plots = True
+            if multi_plots:
```

Comment:
I don't think this if-statement is needed given that you just set `multi_plots` to `True` (so the statement will always pass).